testproject/Assets/Tests/Manual/NetworkAnimatorTests/AnimatedCubeController.cs

```diff
@@ -0,0 +1,34 @@
+using MLAPI;
+using UnityEngine;
+
+namespace Tests.Manual.NetworkAnimatorTests
+{
+    [RequireComponent(typeof(Animator))]
+    public class AnimatedCubeController : NetworkBehaviour
+    {
+        private Animator m_Animator;
+        private bool m_Rotate;
+    
+        private void Awake()
+        {
+            m_Animator = GetComponent<Animator>();
+            m_Rotate = m_Animator.GetBool("Rotate");
+        }
+
+        private void Update()
+        {
+            if (IsOwner)
+            {
+                if(Input.GetKeyDown(KeyCode.C))
```

Comment:
you might consider running `./standards.py --fix` once to format all source files :)
```suggestion
                if (Input.GetKeyDown(KeyCode.C))
```